### PR TITLE
Travis CI: Updates doc info in Adding_CI.rst

### PR DESCRIPTION
**Pull request by Th3nn3ss**
All occurances of the URL was updated
travis-ci.org -> travis-ci.com

Closes https://github.com/coala/coala/issues/6138

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Review by anshalshukla — COMMENTED** [review]
Looks good to me.

File: docs/Developers/Writing_bear_actions.rst [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+            """
+            subprocess.check_call(['git', 'commit', '-o', '--amend'])
+            return file_diff_dict
+
+Note that ``is_applicable`` method is not implemented, that is because
+``ResultAction`` already implements this and it always returns True which
+is what we what want in this case.
+
+``apply`` method spawns a simple process which runs a git command which will
+open up an editor to edit the commit message.
```
Comment (anshalshukla) [inline docs/Developers/Writing_bear_actions.rst:174]: This is not related to this issue.

File: docs/index.rst [hunk 2]
```diff
@@ -50,6 +50,7 @@ If you're trying to **use** coala, you should have a look at our
    Writing Native Bears <Developers/Writing_Native_Bears>
    Writing Linter Bears <Developers/Writing_Linter_Bears>
    Linter Bears - Advanced <Developers/Linter_Bears_Advanced>
+   Writing bear actions <Developers/Writing_bear_actions>
```
Comment (anshalshukla) [inline docs/index.rst:53]: This is also not related.

File: docs/Developers/Adding_CI.rst [hunk 3]
```diff
@@ -36,7 +36,7 @@ forked repository and to do that follow the steps mentioned below.
    <https://github.com/coala/coala/blob/master/.travis.yml>`_.
    Your forked repository from coala will already have that file.
 6. Watch the builds at
-   travis-ci.org/<username>/<repository>/builds.
+   travis-ci.com/<username>/<repository>/builds.
 
 AppVeyor CI
 -----------
```
Comment (anshalshukla) [inline docs/Developers/Adding_CI.rst:42]: These changes look fine.

**Review by anshalshukla — CHANGES_REQUESTED** [review]
Hey, @Th3nn3ss there are some unnecessary changes. You may have a look at my comments on separate files.

File: docs/Developers/Writing_bear_actions.rst (hunk 1 above)
Comment (Th3nn3ss) [reply docs/Developers/Writing_bear_actions.rst:174]: I don't remember editing this files but I will take a look at my code immediately.
